config/root-config.in

```diff
@@ -146,8 +146,8 @@ fi
 ### machine dependent settings ###
 
 case "$cxxversion" in
-  cxx17) cxxversionflag="-std=c++1z " ;;
-  cxx14) cxxversionflag="-std=c++1y " ;;
+  cxx17) cxxversionflag="-std=${cxxversion} " ;;
+  cxx14) cxxversionflag="-std=${cxxversion} " ;;
   *)     cxxversionflag="-std=c++11 " ;;
 esac
```

Comment:
If we are passing down `${cxxversion}` from CMake, then we can get rid of the `case $cxxversion` here. Otherwise, LGTM.